google/cloud/spanner_v1/_helpers.py

```diff
@@ -196,6 +198,152 @@ def _datetime_to_rfc3339_nanoseconds(value):
     return "{}.{}Z".format(value.isoformat(sep="T", timespec="seconds"), nanos)
 
 
+@dataclass
+class Interval:
```

Comment:
In Node , I remember there were a lot of method added to create an Interval type using Months, Days etc.  Eg: 
```
 /**
   * Constructs an `Interval` with specified days.
   */
  static fromDays(days: number): Interval {
    return new Interval(0, days, BigInt(0));
  }
  ```

Reply:
If we want to create that we can do with current code here
`i = Interval(days=number)`
there is no need of exposing extra method here

Reply:
Sounds good. Please if we need documentation for it.